calyx-py/test/correctness/queues/pifo_tree.py

```diff
@@ -1,9 +1,9 @@
 # pylint: disable=import-error
 import sys
 import fifo
-import pifo
 import calyx.builder as cb
 import calyx.queue_call as qc
+import strict_and_rr_queues.gen_strict_or_rr as rr
```

Comment:
I think you should still import it as its original name! `rr` now reads like it's only going to generate round-robin, but of course it can be used to generate strict accidentally!